configs/testing.json

```diff
@@ -0,0 +1,8 @@
+{
+  "defaults": {
+    "files": []
```

Comment:
Don’t we need to deploy some file for testing?

Reply:
I didn't want to include anything go specific in the testing configuration. My thinking was that it can be edited as needed. I added a section about it to the `configs/README.md` -> https://github.com/protocol/.github/blob/a71f87a0d506dfde49763755e91f1eb8732828cb/configs/README.md#testing